src/components/MessagesList/MessagesList.vue

```diff
@@ -49,14 +49,17 @@ get the messagesList array and loop through the list to generate the messages.
 				:count="15" />
 		</template>
 		<transition name="fade">
-			<button v-show="!isChatScrolledToBottom"
+			<Button v-show="!isChatScrolledToBottom"
+				type="secondary"
```

Comment:
Secondary is broken and might not be fixable for 24.

So let's wait with this PR

Reply:
Should we wait another PR ?

Reply:
@marcoambrosini 
Could you please help me for this PR? ;)

Reply:
With the fix from Joas in the vue lib this should be mergeable already! I will release soon!

Reply:
Ok, cool

Reply:
Hi @nickvergessen 
I think it is ready to be reviewed now :)

Reply:
then change the labels :)